lib/symbioticpy/symbiotic/options.py

```diff
@@ -200,6 +200,171 @@ def parse_command_line():
+
+    arg_parser.add_argument('--require-slicer', action='store_true', default=False,
+                            help='Abort if slicing fails/timeouts')
+    arg_parser.add_argument('--full-instrumentation', action='store_true', default=False,
+                            help=('Transform checking errors to reachability problem, i.e.\n'
+                                  'instrument tracking of the state of the program directly\n'
+                                  'into the program.'))
+    arg_parser.add_argument('--bc', action='store_true', default=False,
+                            help='Given files are LLVM bitcode (force this assumption)')
+    arg_parser.add_argument('--undefined-are-pure', action='store_true', default=False,
+                            help='Suppose that undefined functions have no side-effects')
+    arg_parser.add_argument('--undefined-retval-nosym', action='store_true', default=False,
+                            help=('Do not make return value of undefined functions symbolic,\n'
+                                  'but replace it with 0.'))
+    arg_parser.add_argument('--malloc-never-fails', action='store_true', default=False,
+                            help='Suppose malloc and calloc never return NULL')
+    arg_parser.add_argument('--undefines-are-pure', action='store_true', default=False,
+                            help='Suppose that undefined functions have no side-effects')
+    arg_parser.add_argument('--statistics', action='store_true', default=False, help='Dump statistics about bitcode')
+    arg_parser.add_argument('--output', metavar='FILE',
+                            help='Store the final code (that is to be run by a tool) to FILE')
+    arg_parser.add_argument('--witness', metavar='FILE', default='witness.graphml',
+                            help='Store witness into FILE (default is witness.graphml)')
+    arg_parser.add_argument('--working-dir-prefix', default='/tmp',
+                            help='Where to create the temporary directory (defaults to /tmp)')
+    arg_parser.add_argument('--cflags',
+                            help=('Append extra CFLAGS to use while compiling,\n'
+                                  'the environment CFLAGS are used too'))
+    arg_parser.add_argument('--cppflags',
+                            help=('Append extra CPPFLAGS to use while compiling,\n'
+                                  'the environment CPPFLAGS are used too'))
+    arg_parser.add_argument('--argv', metavar='ARGS', help='A comma-separated list of arguments for the main function')
+
+    # # TODO(MichalHe): @review Missing docs.
+    arg_parser.add_argument('--cc', dest='run_in_cc_mode', default=False,
+                            help='')
+
+    # # TODO(MichalHe): @review Docs filled in.
+    arg_parser.add_argument('--link', default='', metavar='LINKFILES',
+                            help='A comma separated list of extra link files.')
```

Comment:
Would be nice to expand the help message with a description of the format these file should be in.